### PR TITLE
Fixes for ClusterLogging and Loki

### PR DESCRIPTION
**Pull request by novacain1**
This PR:

1. Addresses problems observed in resulting Managed clusters where key values were missing from PGT templates (curation).
2. Seeds the workaround secret and certificate for the Yukon Hub cluster in the managed clusters.

### CODE REVIEW

File: clusters/ztp-policies/common/kustomization.yaml [hunk 1]
```diff
@@ -11,6 +11,8 @@ generators:
  # yamllint disable-line rule:line-length
  # This is common to all RAN DU deployments for vRAN Solution Enablement's Synchronization Workgroup
  - common-policies/common-ranGen-vsesync.yaml
+ # This is common to all VSE Loki/Vector clusters
+ - common-policies/common-vseloki.yaml
```
Comment (natifridman) [inline clusters/ztp-policies/common/kustomization.yaml:15]: This should be under resources as this doesn't need to be generated by the plugin

File: clusters/ztp-policies/common/common-policies/common-vseloki.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# yamllint disable rule:line-length
+# for this to take affect, the ManagedCluster must have "vseloki: "true"" as a label
```
Comment (natifridman) [inline clusters/ztp-policies/common/common-policies/common-vseloki.yaml:2]: Can you please explain here what is "this"? Is it this entire file or something else?

**Review by natifridman — CHANGES_REQUESTED** [review]
Looks good, added some comments

File: clusters/ztp-policies/common/kustomization.yaml (hunk 1 above)
Comment (novacain1) [reply clusters/ztp-policies/common/kustomization.yaml]: You are right, fixed.

File: clusters/ztp-policies/common/common-policies/common-vseloki.yaml (hunk 2 above)
Comment (novacain1) [reply clusters/ztp-policies/common/common-policies/common-vseloki.yaml]: Adjusted, see if that makes more sense?

Comment (natifridman) [reply clusters/ztp-policies/common/common-policies/common-vseloki.yaml]: Looks good

**Review by natifridman — APPROVED** [review]

**Pull request merged**